.github/workflows/generate.yml

```diff
@@ -0,0 +1,70 @@
+name: Generate
+
+on:
+  workflow_dispatch:
+    inputs:
+      kubernetesBranch:
+        type: string
+        required: true
+        description: 'The remote kubernetes release branch to fetch openapi spec. .e.g. "release-1.23"'
+      genCommit:
+        type: string
+        required: true
+        default: 'master'
+        description: 'The commit to use for the kubernetes-client/gen repo'
+
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout C
+        uses: actions/checkout@v3
+      - name: Checkout Gen
+        run: |
+          git clone https://github.com/kubernetes-client/gen
+          pushd gen
+          git checkout "${{ github.event.inputs.genCommit }}"
+      - name: Generate Branch Name
+        run: |
+          SUFFIX=$(openssl rand -hex 4)
+          echo "BRANCH=automated-generate-$SUFFIX" >> $GITHUB_ENV
+      - name: Generate Openapi
+        run: |
+          pushd gen/openapi
+          cat <<"EOF"> settings
+          # Kubernetes branch/tag to get the OpenAPI spec from.
+          export KUBERNETES_BRANCH="${{ github.event.inputs.kubernetesBranch }}"
+
+          # client version for packaging and releasing. It can
+          # be different than SPEC_VERSION.
+          export CLIENT_VERSION="0.3.0"
```

Comment:
The `CLIENT_VERSION` is passed to openapi-generator. After the code is generated, it's troublesome to modify. Can we make it configuratable e.g. workflow_dispatch.inputs ?

Reply:
I checked out the code and found the `CLIENT_VERSION` is not used by the C generator.  So we can update it before releasing (Add a manul step to https://github.com/kubernetes-client/c/blob/master/RELEASE.md).

I have no problem here.

Reply:
Fixed this.